daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -765,19 +765,17 @@ private[lf] object SBuiltin {
     }
   }
 
-  /** $tproj[fieldIndex] :: Struct -> a */
-  final case class SBStructProj(fieldIndex: Int) extends SBuiltinPure(1) {
-    override private[speedy] def executePure(args: util.ArrayList[SValue]): SValue =
-      getSStruct(args, 0).values.get(fieldIndex)
-  }
-
   /** $tproj[field] :: Struct -> a */
-  // This is a slower version of `SBStructProj` for the case when we didn't run
-  // the Daml-LF type checker and hence didn't infer the field index.
-  final case class SBStructProjByName(field: Ast.FieldName) extends SBuiltinPure(1) {
+  final case class SBStructProj(field: Ast.FieldName) extends SBuiltinPure(1) {
```

Comment:
In general I am a bit wary of changes which rename B to A at the same time as deleting the old version of A.
But it's not a big deal.

Reply:
In this case we are merging A and B.